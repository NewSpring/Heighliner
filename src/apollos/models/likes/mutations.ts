
export default [
  "toggleLike(contentId: String!): [Like]",
];
