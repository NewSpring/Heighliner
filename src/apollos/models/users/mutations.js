export default [
  "loginUser(email: String!, password: String!): LoginMutationResponse",
  "registerUser(email: String!, password: String!, firstName: String!, lastName: String!): LoginMutationResponse",
  "logoutUser: Boolean",
  "changeUserPassword(oldPassword: String!, newPassword: String!): Boolean",
  "forgotUserPassword(email: String!, sourceURL: String): Boolean",
  "resetUserPassword(token: String!, newPassword: String!): LoginMutationResponse",
  "toggleTopic(topic: String!): Boolean",
];
