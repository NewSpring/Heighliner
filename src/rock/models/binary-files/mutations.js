export default [
  "attachPhotoIdToUser(id: ID!): Boolean",
];
