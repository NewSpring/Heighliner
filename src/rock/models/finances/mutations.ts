
export default [

  `syncTransactions(
    condition: String,
    transaction_type: String,
    action_type: String,
    transaction_id: String,
    order_id: String,
    last_name: String,
    email: String,
    start_date: String,
    end_date: String,
    personId: Int,
    gateway: String = "NMI Gateway",
  ): [Transaction]`,

];
