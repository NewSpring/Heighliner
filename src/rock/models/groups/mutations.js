
export default [
  `requestGroupInfo(
    groupId: ID!,
    message: String!,
    communicationPreference: String!,
  ): GroupsMutationResponse`,
];
