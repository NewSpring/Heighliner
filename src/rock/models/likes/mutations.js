export default ["toggleLike(nodeId: String!): LikesMutationResponse"];
