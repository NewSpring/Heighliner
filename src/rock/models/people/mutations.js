export default [
  "setPhoneNumber(phoneNumber: String!): PhoneNumberMutationResponse",
  "setPersonAttribute(attributeId: Integer!, attributeValue: String!): AttributeValueMutationResponse",
];
