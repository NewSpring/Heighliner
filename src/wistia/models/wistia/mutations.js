export default [
  "createProject(name: String!): WistiaProject",
  "createMedia(name: String!, description: String, url: String!, project_id: String!): WistiaMedia",
];
