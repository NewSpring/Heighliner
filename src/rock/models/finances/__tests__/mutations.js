
import mutations from "../mutations";

it("should have all needed mutations", () => {
  expect(mutations).toMatchSnapshot();
});
