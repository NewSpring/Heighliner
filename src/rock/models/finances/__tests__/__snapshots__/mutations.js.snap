exports[`test should have all needed mutations 1`] = `
Array [
  "syncTransactions(
    condition: String,
    transaction_type: String,
    action_type: String,
    transaction_id: String,
    order_id: String,
    last_name: String,
    email: String,
    start_date: String,
    end_date: String,
    personId: Int,
    gateway: String = \"NMI Gateway\",
  ): [Transaction]",
  "createOrder(
    data: String!
    id: ID
    instant: Boolean = false,
    gateway: String = \"NMI Gateway\",
    url: String
  ): OrderMutationResponse",
  "completeOrder(
    token: ID!
    scheduleId: ID
    accountName: String
    gateway: String = \"NMI Gateway\",
  ): CompleteOrderMutationResponse",
  "validate(
    token: ID!
    gateway: String = \"NMI Gateway\",
  ): ValidateMutationResponse",
  "cancelSavedPayment(
    id: ID
    entityId: Int
    gateway: String = \"NMI Gateway\",
  ): SavePaymentMutationResponse",
  "savePayment(
    token: ID!
    accountName: String
    gateway: String = \"NMI Gateway\",
  ): SavePaymentMutationResponse",
  "updateSavedPayment(
    entityId: Int
    name: String!
  ): SavePaymentMutationResponse",
  "cancelSchedule(
    id: ID
    entityId: Int
    gateway: String = \"NMI Gateway\",
  ): ScheduledTransactionMutationResponse",
  "transactionStatement(
    limit: Int
    skip: Int
    people: [Int]
    start: String
    end: String
  ): StatementMutationResponse",
]
`;
